app/src/main/res/values/colors.xml

```diff
@@ -38,4 +43,15 @@
     <color name="start_gradient">#A173FF</color>
     <color name="finish_gradient">#FC203E</color>
 
-</resources>
\ No newline at end of file
+    <!-- Color UI -->
+    <color name="primary">#008577</color>colorPrimary
```

Comment:
In colors color name should describe color itselt but not used place. If you wanna use extra name for specific reason, add it with a link to normally called color. 

Also, here you uses naming, that are common for themes. Suppose, it could be a problem with real theme usage.